port_forwarding.go

```diff
@@ -324,12 +326,12 @@ func (pf *PortForwarder) runTunnel(ctx context.Context, remote net.Conn) {
 	done := make(chan struct{}, 2)
 
 	go func() {
-		io.Copy(local, remote)
+		_, _ = io.Copy(local, remote)
```

Comment:
suggestion: If this is to silence the errcheck linter I'd suggest using the `nolint` directive instead.
```suggestion
		io.Copy(local, remote) //nolint:errcheck // Nothing we can usefully do with the error
```